### PR TITLE
Add remediation metadata to the generated playbooks

### PR DESCRIPTION
**Pull request by jan-cerny**
#### Description:
This will add the remedaition metadata in form of comments s.a.
```
# reboot = false
...
```
because are not present in generated per-rule playbooks.

Also adds reboot tag to generated snippets.

#### Rationale:
Fixes: #4194

### CODE REVIEW

File: ssg/ansible.py [hunk 1]
```diff
@@ -100,6 +100,8 @@ def update_tags_from_config(self, to_update):
             tags.append("{0}_complexity".format(self.config["complexity"]))
         if "disruption" in self.config:
             tags.append("{0}_disruption".format(self.config["disruption"]))
+        if "reboot" in self.config:
+            tags.append("{0}_reboot".format(self.config["reboot"]))
```
Comment (matejak) [inline ssg/ansible.py]: `false_reboot` / `true_reboot` are somehow confusing, what about `reboot_required` / `no_reboot_needed`?

Comment (jan-cerny) [reply ssg/ansible.py]: yes, thank you for the suggestion

Comment (jan-cerny) [reply ssg/ansible.py]: @matejak changed

**Comment by matejak** [comment]
Works great, merging!

**Pull request merged**